thoth/storages/graph/postgres.py

```diff
@@ -4047,7 +4047,7 @@ def delete_kebechet_github_app_installations(
         runtime_environment_name: Optional[str] = None,
         info_manager: Optional[bool] = None,
         pipfile_requirements_manager: Optional[bool] = None,
-        udpate_manager: Optional[bool] = None,
+        update_manager: Optional[bool] = None,
```

Comment:
same change is also needed at https://github.com/thoth-station/storages/blob/f3b00194bcb61eac2573ed2f874ffe17bb6167e7/thoth/storages/graph/postgres.py#L3955

Reply:
nice catch! fixed!